.gitattributes

```
@@ -5,6 +5,7 @@
 
 # Don't diff machine generated files
 package-lock.json  -diff
+Gemfile.lock  -diff
```

Minor nitpick, but given that everything is aligned here can you align this one as well? :sweat_smile: 

```suggestion
package-lock.json  -diff
Gemfile.lock       -diff
```